src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenForClientParameterBuilder.cs

```diff
@@ -122,6 +124,28 @@ public AcquireTokenForClientParameterBuilder WithPreferredAzureRegion(bool useAz
             throw new NotImplementedException();
         }
 
+        /// <summary> 
+        /// Adds an fmi_path parameter to the request. It changes the subject of the token. 
+        /// </summary>
+        public AcquireTokenForClientParameterBuilder WithFmiPath(string pathSuffix)
+        {
+            ValidateUseOfExperimentalFeature();
+
+            if (string.IsNullOrEmpty(pathSuffix))
+            {
+                throw new ArgumentNullException(nameof(pathSuffix));
+            }
+
+            var cacheKey = new SortedList<string, string>();
+            cacheKey.Add(OAuth2Parameter.FmiPath, pathSuffix);
```

Comment:
nit: VS suggests you to simplify this: 

 var cacheKey = new SortedList<string, string>
 {
     { OAuth2Parameter.FmiPath, pathSuffix }
 };